pages/osx/lp.md

```diff
@@ -0,0 +1,15 @@
+# lp
+
+> print files
+
+- print a file to a named printer (found in Printers & Scanners --> Options and Supplies)
+
+'lp -d {{device_name}} {{path/to/filename}}
```

Comment:
Again, that is probably an OSX-specific feature that doesn't translate to other systems. Besides, I don't think this would be confusing if instead of instructions to look at the printers using the GUI, we suggest using `lpstat`.